src/neuroglancer/chunk_manager/base.ts

```diff
@@ -35,9 +34,23 @@ export enum ChunkState {
   // Download failed.
   FAILED = 6,
 
-  EXPIRED = 7
+  EXPIRED = 7,
+
+  COMPUTING = 8
 }
 
+export const CHUNK_STATE_NAMES = [
```

Comment:
The TypeScript compiler actually already provides this: ChunkState[ChunkState.COMPUTING] === 'COMPUTING'